app/components/live-header.hbs

```diff
@@ -0,0 +1,8 @@
+<section class='live__header'>
+  <Reusables::Button
+    @variant='share'
```

Comment:
button variant can be only three according to reusable button
![image](https://user-images.githubusercontent.com/85296770/230651652-bf6e74ae-6068-405c-add8-d1b6c6f9885b.png)

Reply:
When we make components reusable, we don't restrict them. It will be like a component is reusable but you only need to use it in these given variants. These don't sound good to me. Please correct me if I am wrong.